### DISCUSSION TITLE
How to view the output of Kepler?

### DISCUSSION THREAD

**Discussion by nraxana** [sustainable-computing-io/kepler#1287] [Q&A]
Hi Team,

I have deployed Kepler using Helm chart and also deploy kube-prometheus for viewing output in Kubenetes cluster.
However I am not getting any response using the "kubectl -n monitoring port-forward svc/grafana 3000" command.
It shows as "Forwarding..." as attached in image but does not expose the web page UI.
Kindly help in how to access the dashboards, Thank you.
![Grafana](https://github.com/sustainable-computing-io/kepler/assets/162302355/f93d83cd-497b-46a2-81a6-eba8497a6ca1)

**Comment by marceloamaral** [comment]
@nraxana this is not a Kepler related question. but how to install Grafana :)
Have you tried to verify what is going on with the grafana pod?
`kubectl get pods -n monitoring`
`kubectl logs -n monitoring service/grafana`

**Reply by nraxana** [reply]
Thank you for the reply @marceloamaral. I have tried to verify and below is the output.

![kubectl get pods -n monitoring](https://github.com/sustainable-computing-io/kepler/assets/162302355/7c8eccd3-14f5-41af-a40e-c64e56709795)

![kubectl logs -n monitoring servicegrafana](https://github.com/sustainable-computing-io/kepler/assets/162302355/40668017-bd40-44b2-b9f4-4bbc1a71bc6e)

Kindly let me know how to proceed further.

Thank you in advance.
